src/handler.test.js

```diff
@@ -381,3 +382,25 @@ test("Response with malformed JSON returns an error", async () => {
     expect(result.body).toContain('"message":"An error has occurred."');
     expect(result.body).toContain('"timestamp":');
 });
+
+test("Test the numberOfMatches value returned", async () => {
```

Comment:
The convention with these test names is to use the `test` to start a sentence explaining what the test does.
```suggestion
test("the numberOfMatches value is returned", async () => {
```